internal/cli/cli.go

```diff
@@ -70,3 +99,17 @@ func pluginTypeSetup() error {
 
 	return nil
 }
+
+// pflag StringSlice() delimits strings by comma but viper GetStringSlice() does not.
+// https://github.com/spf13/viper/issues/380
+// This causes issues for environment variables, example: ENV=test1,test2.
+// Delimit the input by commas and return the result.
+func delimitByComma(stringSlice []string) []string {
```

Comment:
Nit: cleanEnvVarSlice would be clearer to me but the input PLUGIN_COMMANDS="version manifest,version" yields the correct output.

Reply:
Thanks, renamed to `cleanEnvVarSlice`